wiki/Modding/en.md

```diff
@@ -1,98 +1,237 @@
+  Kudosu can be used to boost a map's [star priority](/wiki/star_priority).
+- "M4M" (mod-for-mod); when a modder makes a deal to mod the map if the receiver mods the modder's map.
+  This is a great and easier way to get maps modded.
+- A good mod post can help the approval teams from spending their time from pointing out basic mistakes.
+  This in turn will help the approval teams reduce their stress and may become more likely to mod the map.
 
-Basic modding method
---------------------
+Whether you are learning or already know how to map, modding is a great way to help improve your mapping skills and the mapping/modding community.
 
-First, download a map from the [beatmap](beatmap "wikilink") thread, which can be located in either the [Pending](Pending_Beatmap "wikilink") or [Work In Progress/Help](Works_In_Progress/Help "wikilink") forum (not the [Ranked](Ranked_beatmap "wikilink")/[Approved](Approved_Beatmap "wikilink") Beatmaps forum!). You can either randomly choose maps from these sub-forums to mod or you can open a modding queue in the **Modding Queue** sub-forum under the [Work In Progress/Help](Works_In_Progress/Help "wikilink") forum and accept requests. If you are opening a queue, it is a good idea to see how these queues work by reading some threads in the 'Modding Queues' sub-forum.
+## Getting Started
 
-Next, go into the [Editor](Editor "wikilink") and select the map which you want to mod. Then, start up a reply to the [Beatmap's](Beatmap "wikilink") thread.
+To get started, go to the [Pending](https://osu.ppy.sh/forum/6) or [Work In Progress/Help](https://osu.ppy.sh/forum/10) subforums.
+From here, you can either randomly choose beatmaps from these subforums to mod or open a modding queue in the [Modding Queue](https://osu.ppy.sh/forum/60) subforum and accept requests.
+If you are opening a queue for the first time, it is a good idea to see how modding queues work by looking at some of the existing threads in the Modding Queue subforum.
 
-Choose any of the difficulties to begin your mod. It is recommended to start with the easiest or hardest difficulty and make your way up or down the different difficulties.
+Open the beatmap's forum thread by clicking on `Web` then `This Beatmap's Thread`.
+From here, click the `Reply` button on the bottom-left to start your mod post.
+In this post, you should add a heading to denote the difficulty(ies) you are modding.
 
-Start by playing this difficulty in either Play mode or [test mode](Glossary#Test_Mode "wikilink"). Play-testing is an important part of modding as it allows the modder to understand whether or not certain [patterns](Mapping_Techniques "wikilink") play well. Take note of anything that felt odd.
+Next, open the song in the osu!client.
+You can choose any of the difficulties to start a mod but it is recommended to start with the easiest difficulty and continue with harder difficulties or vise versa.
```

Comment:
I'm not fully sure... (this was translated over from the old article). Change wording to:
`You can choose any of the difficulties to start a mod.`
❓